networkpolicy/bedrock-access-to-ibm-common-service-operator.yaml

```diff
@@ -0,0 +1,15 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: access-to-ibm-common-service-operator
```

Comment:
Yes, as Daniel said https://github.ibm.com/IBMPrivateCloud/roadmap/issues/56443#issuecomment-51576632 when `deny-all` ingress and egress policies take effect in the namespace, it will block the communications to webhook under cs-operator. This new network policy is to unblock it.

Reply:
Yes, allows inbound traffic to cs-operator because the webhook process is running inside it now.